tasks/main.yml

```diff
@@ -53,26 +53,37 @@
     is_prod: "{{ archivematica_src_environment_type == 'production' }}"
   tags: "always" # inocuous to use always here
 
-- name: "Include common-RHEL.yml for RHEL"
-  include: "common-RHEL.yml"
-  when: "ansible_distribution == 'RedHat'"
 
-- name: "Install necessary packages required by this role"
-  package:
-    name: "{{ item }}"
-    state: "latest"
-  with_items: "{{ ansible_deps }}"
+- name: "Configure common prerequisites"
+  block:
 
-- name: "Include common.yml common tasks for all components"
-  include: "common.yml"
-  tags: # do not use "always" tag in role, to avoid issues when including other roles in a playbook
-    - "amsrc-common"
-    - "amsrc-ss"
-    - "amsrc-pipeline"
-    - "amsrc-devtools"
-    - "amsrc-automationtools"
-    - "amsrc-fixity"
+  - name: "Include common-RHEL.yml for RHEL"
+    include: "common-RHEL.yml"
+    when:
+      - "ansible_distribution == 'RedHat'"
+
+  - name: "Install necessary packages required by this role"
+    package:
+      name: "{{ item }}"
+      state: "latest"
+    with_items: "{{ ansible_deps }}"
```

Comment:
We should stop using `with_items` (deprecated) in the apt, yum and packages modules whenever possible. In the case of osdeps where the state is included in the dict, it is more complicated to do, but for these simple cases we should use:

```
  - name: "Install necessary packages required by this role"
    package:
      name: "{{ ansible_deps }}"
      state: "latest"
``` 

And when using a explicit list of packages:

``` 
   - name: "Install necessary packages required by rpms"
     yum:
       name:
         - "python-virtualenv"
         - "python-pip"
       state: "present"
```

Instead of:

```
   - name: "Install necessary packages required by rpms"
     yum:
       name: "{{ item }}"
       state: "present"
     with_items:
       - "python-virtualenv"
       - "python-pip"
``` 

We can change it here or make a separate PR.